### DISCUSSION TITLE
remove ship methode

### DISCUSSION THREAD

**Discussion by tomasdecastro** [woocommerce/woocommerce#53321] [Checkout Flow]
Who can i remove ship methode from check out page in a salient woocommerce store

**Comment by nielslange** [comment]
Hello @tomasdecastro 👋

Thanks for reaching out. To answer your question, we'd need additional information. What shipping method are you trying to remove? Should this shipping method be removed for all locations or only for certain locations? What do you mean by _"a salient woocommerce store"_?
